cleantalk.php

```diff
@@ -1341,6 +1331,51 @@ function apbct_sfw_update__init($delay = 0)
     );
 }
 
+/**
+ * Decide need to force direct update
+ *
+ * @return bool
+ * @psalm-suppress NullArgument
+ */
+function apbct_sfw_update__switch_to_direct()
+{
+    global $apbct;
+
+    $apbct->fw_stats['reason_direct_update_log'] = null;
```

Comment:
Please, add this array key `reason_direct_update_log` in the default `fw_stats` array contained in the `State` class

Reply:
https://github.com/CleanTalk/wordpress-antispam/pull/350/commits/707cafe7f4da371bf0126633bae0dcc8a0b938ea